basculechecks/basculechecks.go

```diff
@@ -107,6 +117,30 @@ func NewCapabilityChecker(m *AuthCapabilityCheckMeasures, prefix string, acceptA
 	return &c, nil
 }
 
+// NewCapabilityCheckerFromStrings creates the capability checker, and allows
+// consumers to provide a list of string endpoints, rather than regular
```

Comment:
Maybe rephrase to something like: 
```
// NewCapabilityCheckerFromStrings creates the capability checker from a list of endpoint regular expressions as 
// strings to be compiled. Only expressions that compile successfully are included in the checker.
```
To be explicit about how the strings will be compiled to a re and that only the successfully compiled ones will make the final check list.